.pre-commit-config.yaml

```diff
@@ -13,6 +13,8 @@ repos:
             app/modules|
             app/components
           )
+        args:

```

Comment:
Did you want to add this?

Reply:
Yeah it was intentional, I am not a terraform expert but someone who is has recommended it. It will stop the issue where we need to keep doing `terraform init` in all the folders when committing. It should automatically do that once and retry.